_includes/js/timeline-js.html

```diff
@@ -6,10 +6,20 @@
         return new bootstrap.Tooltip(tooltipTriggerEl)
     })
 
-    // highlight year if hash
-    if(window.location.hash) {
-        var hashfilter = decodeURIComponent(location.hash.substr(1));
-        document.querySelector("tr#" + hashfilter).classList.add("table-info");
-    }
+    // Highlight years based on query parameters
+    var queryParams = new URLSearchParams(window.location.search);
```

Comment:
I think you have hash-based routing and query parameter messed up. The URLs generated are wrong

`/timeline.html?y1920y1965#y1920`

Reply:
You are right. My bad.